client/server/api/v1/rfc/[rfcId].ts

```diff
@@ -13,12 +13,12 @@ export default defineEventHandler(async (event) => {
   const { rfcId } = params
   const rfcPatterns = rfcId.match(rfcRegex)
   if (!rfcPatterns) {
-    throw Error(`Expected param to match ${rfcId}`)
+    throw Error(`Expected param to be a valid RFC id JSON`)
```

Comment:
I'm a bit puzzled by the phrase "RFC id JSON" - I think I know what it's getting at but I'm not sure why the phrase was chosen. Does this error propagate somewhere that it might reach a user's eyes? If so, something easier to parse / understand; if not, it's ok

Reply:
Yeah I wasn't sure about that error message. It's whether part of the URL matches the `rfcRegex` which is basically `rfc*.json` but regexy.

I'll change it to `invalid URL` or something.